scale-testing.mk

```diff
@@ -5,13 +5,19 @@
 AWS_REGION ?= us-east-2
 EKS_K8S_VERSION ?= 1.30
 
+VERSION ?= 0.8.1
+INTEGRATION_TESTS_PARALLEL ?= true
+SKIP_HCPVSAPPS_TESTS ?= true
```

Comment:
Same for this one. The default should be false.

Reply:
The default is false, but the tests will fail if these are not skipped since it's setup to use KIND.

Reply:
Maybe move it to the main Makefile and set its value to `false` by default